### DISCUSSION TITLE
Milestone4 due Monday!!

### DISCUSSION THREAD

**Discussion by inorrr** [CSC207-2022F-UofT/course-project-macrosoft#56] [Announcements]
Milestone4 is due Monday after the lab, so make sure you have completed at least a significant amount of code for your part. Also remember to take a look at other members part and pick two pull requests: 

> One demonstrating most significant code contribution(writing code)
> One demonstrating most signifiant code review(giving feedback to another member)

For each of these pull requests, include a brief explanation of why you think it was significant. Ideally, your team should discuss these and take the opportunity to highlight how your project has satisfied the rubric elements. You don't have to explicitly touch on all elements of the rubric, but anything you highlight here will help ensure your TA is aware of all the great work your team has been doing! 

More details please refer to Milestone 4 at -> https://q.utoronto.ca/courses/278453/pages/milestone-4-implementation?module_item_id=4109691

**Comment by xuallenx** [comment]
Thanks for the reminder yinuo!

**Comment by leoyliu** [comment]
UPPP!

**Comment by KaiwenYangUT** [comment]
Let's go! We can do this!
